indico/util/spreadsheets.py

```diff
@@ -128,7 +128,9 @@ def _prepare_excel_data(data, tz=None):
     elif is_lazy_string(data) or isinstance(data, Markup):
         data = str(data)
     elif isinstance(data, datetime):
-        data = format_datetime(data, timezone=tz)
+        data = format_datetime(data, format='yyyy/MM/dd HH:mm', timezone=tz)
```

Comment:
getting some weird results with the Hawaii timezone, which has a 30min offset. the offset is being lost somehow. maybe outdated pytz or babbel? very weird regardless